src/Entity.js

```diff
@@ -214,22 +259,9 @@ exports = Class(function() {
 	 *         ~ returns the total distance moved by the two entities
 	 */
 
-	this.collidesWith = function(entity) {
-		return this.physics.collide(this, entity);
-	};
-
-	this.resolveCollidingStateWith = function(entity) {
```

Comment:
I did some fixes on `merge-sat` branch and also formatted the code style. Thank @rogueSkib !